content/hardware/ordering.md

```diff
@@ -0,0 +1,101 @@
+date: 2018-11-28T15:14:54+10:00
+intro_image: imgs/single-car.jpg
+image: "/services/default.png"
+featured: true
+draft: false
+active: true
+summary: Pick and Order your own MuSHR Racecar!
+weight: 1
+---
+
+This section explains the MuSHR v4 tier system, allowing you to order and build a version of the MuSHR car that best meets your needs.
+
+___
+## Table of Contents
+
+1. [System Overview](#system-overview)
+2. [Add-on Modules](#add-on-modules)
+3. [Recommended Tier System](#recommended-tier-system)
+4. [Custom MuSHR tiers](#custom-mushr-tiers)
+5. [3D Printing](#3d-printing-files)
+
+___
+## System Overview
+
+This section describes the high level system architecture of the MuSHR racecar and your available options to build the car's hardware. Check out the [build tutorial](/hardware/build_instructions) to get an idea of what building the car is like!
+
+**Processor**
+
+The MuSHR Racecar can be built with a choice of two main processors, the Nvidia Jetson Nano or the Nvidia Jetson Xavier NX. The latter is better for high-performance applications, while the former is better if you're working on a budget. You can view a comparison of the two [here.](https://www.seeedstudio.com/blog/2020/06/04/nvidia-jetson-nano-and-jetson-xavier-nx-comparison-specifications-benchmarking-container-demos-and-custom-model-inference/)
+
+**Sensing**
+
+MuSHR supports the Intel RealSense D435/D435i and the D455 cameras for depth perception. The car also has a scanning YDLIDAR X4, and other optional add-on sensors listed below. View a comparison of the different Intel RealSense cameras [here.](https://www.intelrealsense.com/compare-depth-cameras/)
+
+**Chassis**
+
+The base chassis of the car is a Redcat Racing Rally Car with two motor options, modified extensively with 3D printed parts. The Pro chassis is more expensive, but follows a simpler build process, not requiring motor replacement, saving hours of build time.
+
+___
+## Add-on Modules 
+
+You can also pick add-on modules to supplement your car and augment its capabilities to perform certain functions.
+
+**Push Button**
+
+A button mounted to the front of the car, allowing you to detect if the front bumper is in contact with an object or surface.
+
+**T265 Tracking Camera**
+
+The MuSHR Racecar can have a second camera mounted to it, the Intel RealSense T265. This camera is particularly useful for tracking.
+
+**Radio FPV**
+
+If you need to tele-operate outdoors or over a longer distance outside standard Bluetooth/WiFi range, you can follow the steps in [this tutorial](/tutorials/radio_fpv) to add a radio-based teleoperation system to your racecar.
+
+**Pushing Bumper**
+
+You can mount a 3D-Printed bumper to the front of the car to perform object manipulation tasks with the car.
+
+___
+## Recommended Tier System
+
+As a result of the various options available for MuSHR, we have simplified the MuSHR Racecar into three different tiers, illustrated in the table below. 
```

Comment:
Add a sentence about how these tiers do not need to be followed and that any combination of cpu, camera, chassis will work. Good to over emphasize because someone may miss that above